src/Icons/doc/index.rst

```diff
@@ -166,7 +175,7 @@ HTML Syntax
 
 In addition to the ``ux_icon()`` function explained in the previous sections,
 this package also supports an alternative HTML syntax based on the ``<twig:ux:icon>``
-tag:
+tag if the ``symfony/ux-twig-component`` package is installed:
```

Comment:
<img width="689" alt="image" src="https://github.com/user-attachments/assets/47974500-86c7-49b6-8e71-5d75167bb59a" />

Does feel like a duplicate of the tip 10 lines below. 

What about moving it after the introduction sentence ? and before the code examples ?

Reply:
i duplicated it on purpose, becaues it is very important, and i think some people will rather see the box, while others rather read the text flow. alternatively we could even put it into the code block as a comment right before the first tag.

which introduction sentence do you mean? the "HTML Syntax" section only has one sentence until here? or you mean moving the `..tip` to above the code? i can also do that.